fpga.chart.py

```diff
@@ -200,7 +197,7 @@ def _get_fpga_power(self, idx):
     def get_fpga_os_status(self):
         result_dict = {'result': '0'}
         while True:
-            time.sleep(2)
+            time.sleep(self.temp_power_update_interval - 2)
```

Comment:
Why 2?

Reply:
The time it takes to run the command is 1.4 seconds for Intel. Xilinx is much quicker.

Reply:
This can be done when I change the way the cards are detected as mentioned above rather than as part of this task.

Reply:
As above, 2 needs to be calculated. Should not be determined by which card is used.